src/screens/FeedScreen/hooks/useCoreFeed.js

```diff
@@ -214,18 +168,6 @@ const useCoreFeed = () => {
     }
   }, [isLastPage, isScroll]);
 
-  const getRandomInt = (min, max) => {
-    // Create byte array and fill with 1 random number
-    const byteArray = new Uint8Array(1);
-    // eslint-disable-next-line no-undef
-    crypto.getRandomValues(byteArray);
-
-    const range = max - min + 1;
-    const max_range = 256;
-    if (byteArray[0] >= Math.floor(max_range / range) * range) return getRandomInt(min, max);
-    return min + (byteArray[0] % range);
-  };
-
   const sendViewPostTime = async (withResetTime = false) => {
     sendViewPostTimeWithFeeds(feeds, withResetTime);
   };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Removing the `getRandomInt` function and not using it elsewhere in the code is a good move if it's no longer needed. It reduces unnecessary complexity. However, make sure that this function is not used anywhere else in your project before removing it.
```diff
- const getRandomInt = (min, max) => {...};
+ // No replacement needed
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->